docs/source/examples/BSTLD.rst

```diff
@@ -174,106 +166,83 @@ structure for ``BSTLD`` should be like:
 
 .. _graviti opendatset platform: https://www.graviti.cn/open-datasets
 
-TensorBay supplies two methods to fetch :ref:`basic_concepts:Segment` from
-:ref:`basic_concepts:Dataset`.
-
-.. code:: python
+TensorBay supplies two methods to fetch :ref:`reference/dataset_structure:Segment` from
+:ref:`reference/dataset_structure:Dataset`.
 
-    >>> train_segment = dataset.get_segment_by_name("train")
-
-    >>> first_segment = dataset[0]
-
-The :ref:`basic_concepts:Segment` you get now is the same as the one you read from TensorBay in the
-:ref:`quick_start:Read Dataset from TensorBay` part.
+.. literalinclude:: ../../../examples/bstld.py
+   :language: python
+   :lines: 20-21
 
-**************************
- Write Dataset Dataloader
-**************************
+The :ref:`reference/dataset_structure:Segment` you get now is the same as the one you read from TensorBay in the
+:ref:`quick_start/getting_start_with_tensorbay:Read Dataset from TensorBay` part.
 
-If there is no :ref:`contribution:Dataloader` available to your target dataset,
-you can write one and contribute it.
-Now we take ``BSTLD`` as an example to explain how to write a dataloader for datasets with Box2D
-labels.
+*****************************
+ Upload images to the Dataset
+*****************************
 
-Write the Catalog
-=================
+Once you write your own :ref:`reference/glossary:Dataloader` and read the local dataset into a
```

Comment:
After you finish the :ref:`reference/glossary:Dataloader` and organize the BSTLD into a ``Dataset`` class,